code/modules/atmospherics/machinery/components/gas_recipe_machines/crystallizer_items.dm

```diff
@@ -0,0 +1,33 @@
+/obj/item/hypernoblium_crystal
+	name = "Hypernoblium Crystal"
+	desc = "crystalized oxygen and hypernoblium stored in a bottle to pressureproof your clothes."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "potblue"
+	var/uses = 2
+
+/obj/item/hypernoblium_crystal/afterattack(obj/item/clothing/C, mob/user, proximity)
```

Comment:
Please use informative names, not just a letter.

Reply:
As per https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names